optimera-htb-exports.js

```diff
@@ -11,3 +11,8 @@ shellInterface.OptimeraHtb = {
     render: SpaceCamp.services.RenderService.renderDfpAd.bind(null, 'OptimeraHtb')
 };
 //? }
+
+if (__directInterface.Layers.PartnersLayer.Partners.OptimeraHtb) {
```

Comment:
This part is not necessary as your request type is Ajax and does not use a callback function

Reply:
Ok, will take out.